src/java/pom.ts

```diff
@@ -148,6 +149,56 @@ export interface MavenRepository {
    * The layout of the repository
    */
   readonly layout?: string;
+  /**
+   * Repository Policy
+   */
+  readonly snapshots?: MavenRepositoryPolicy;
+  readonly releases?: MavenRepositoryPolicy;
+}
+
+/**
+ * Represents a Maven Repository Policy
+ * @see https://maven.apache.org/settings.html#repositories
+ */
+export interface MavenRepositoryPolicy {
+  /*
+   * Is enabled
+   */
+  readonly enabled?: boolean;
+
+  /**
+   * Update Policy
+   */
+  readonly updatePolicy?: UpdatePolicy;
+
+  /**
+   * Checksum Policy
+   */
+  readonly checksumPolicy?: ChecksumPolicy;
+}
+
+export class UpdatePolicy {
+  static get always() {
```

Comment:
By the way, these don't need to be getters.
```suggestion
  static get ALWAYS() {
```